pages/press.md

```diff
@@ -629,9 +629,6 @@ Are you looking to get a speaker or interview about Solid? Contact us on press@s
 
 2019-07-31 [The Verge: Brain-computer interfaces are developing faster than the policy debate around them](https://www.theverge.com/interface/2019/7/31/20747916/facebook-brain-computer-interface-policy-neuralink)
 
```

Comment:
There's an archived version of this that we can link to:

```suggestion

2019-07-31 [Joy Online: Apple joins Google, Facebook, and Twitter in data-sharing project](https://web.archive.org/web/20191016140751/https://myjoyonline.com/technology/2019/july-31st/apple-joins-google-facebook-and-twitter-in-data-sharing-project.php)
```

(For some reason GitHub won't let me make the suggestion on the deleted line, so it looks somewhat weird.)